### PR TITLE
docs(agw): Include an FAQ on how to check subscriber is idle of conne…

### PR DESCRIPTION
**Pull request by joary**
…cted

Signed-off-by: Joary Paulo Wanzeler Fortuna <joarypl@fb.com>

## Summary

Included an FAQ on how to check if a given subscriber is IDLE of CONNECTED

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/faq/faq_magma.md [hunk 1]
```diff
@@ -103,6 +103,14 @@ This section lists some of the commonly asked questions related to Magma operati
 
 - <https://github.com/magma/magma/tree/master/docs/readmes/lte>
 
+### How can I check if a given subscriber is idle or connected?
+
+  - With the users IMSI you can fetch the state of a given subscriber:
```
Comment (andreilee) [inline docs/readmes/faq/faq_magma.md]: Include detail that you need to SSH into the gateway where the subscriber is connected.

**Review by andreilee — APPROVED** [review]

File: docs/docusaurus/versioned_docs/version-1.5.0/faq/faq_magma.md [hunk 2]
```diff
@@ -86,6 +86,13 @@ This section lists some of the commonly asked questions related to Magma operati
 ### Where can I get more information about services running in Magma?
   - https://github.com/magma/magma/tree/master/docs/readmes/lte
 
+### How can I check if a given subscriber is idle or connected?
+  - With the users IMSI you can fetch the state of a given subscriber:
+    - `state_cli.py parse IMSI716770120257790:MME`
+  - looking at the `ecm_state` configuration, the value:
```
Comment (hcgatewood) [inline docs/docusaurus/versioned_docs/version-1.5.0/faq/faq_magma.md]: Nit

```suggestion
  - Looking at the `ecm_state` configuration, the value:
```

**Review by hcgatewood — APPROVED** [review]
Lgtm! Incorporate Andrei's comment, and, if we're going to backport this to the 1.5 docs, also add it to the 1.6 docs

**Comment by ekowtaylor** [comment]
@joary do you want to rebase and see through merging this PR?

**Pull request merged**